composer.json

```diff
@@ -9,7 +9,7 @@
         {"name": "Steve Lacey", "email": "steve@steve.ly"}
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^7.4 || ^8.0",
```

Comment:
@maikelohcfg Are you sure? I still see this line in my browser.

Reply:
Ups not committed properly, now is done :).

Reply:
Sorry

Reply:
Thank you